.github/workflows/build.yml

```diff
@@ -46,3 +46,4 @@ jobs:
             "all gatlingScalafixCheck" \
             "test" \
             "scripted"
+        shell: bash
```

Comment:
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#defaultsrun